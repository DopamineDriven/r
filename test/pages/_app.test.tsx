import React, { Component } from 'react';
import NextApp from '../../pages/_app';
import { AppProps } from 'next/app';
import { Router } from 'next/router';
import { mocked } from 'ts-jest/utils';
let router: Router;
let pageProps: AppProps;

describe('App root', () => {
	it('matches snapshot', () => {
		const AppMocked = mocked(
			<NextApp
				Component={Component}
				pageProps={pageProps}
				router={router}
			/>
		);
		expect(AppMocked).toMatchSnapshot();
	});
});
