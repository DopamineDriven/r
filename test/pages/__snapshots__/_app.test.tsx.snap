// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App root matches snapshot 1`] = `
<NextApp
  Component={[Function]}
/>
`;
